internal/k8s/exec.go

```diff
@@ -34,6 +35,9 @@ type ExecParameters struct {
 	Pod       string
 	Container string
 	Command   []string
+	TTY       bool
```

Comment:
No need for this to be configurable for now IMO. We're not going to open actual TTYs in this tool, so best to omit it and make `(*Client).execInPod` opinionated. Keep it simple!

Reply:
Removed, always passing TTY as `true`.